compiler/next/lib/resolution/resolution-queries.cpp

```diff
@@ -403,16 +403,20 @@ struct Resolver {
         const BorrowedIdsWithName& m = vec[0];
         if (m.id(0) == decl->id() && m.numIds() > 1) {
           Location loc = parsing::locateId(context, decl->id());
+          // TODO: Overload of 'context->error()' that can handle these.
```

Comment:
Not so sure that is possible, if we want the framework to be able to be aware of errors that are grouped